app/templates/email/video_stream_moderator.html

```diff
@@ -0,0 +1,5 @@
+{{ _('Hi you have been added as a video moderator in the event') }} {{ video_stream_name }}.
```

Comment:
Please change the message to what was discussed in chat. Both video name and event should be listed

Reply:
But that was discussed in subject

Reply:
Video Moderator of video {video_name} at event {event_name}

Reply:
@iamareebjamal done the changes